target/linux/ramips/dts/DWR-922-E2.dts

```diff
@@ -0,0 +1,152 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "dlink,dwr-922-e2", "ralink,mt7620n-soc";
+	model = "D-Link DWR-922 E2";
+
+	aliases {
+		led-boot = &led_sstrenghg;
+		led-failsafe = &led_sstrenghg;
+	};
+
+	gpio-keys-polled {
```

Comment:
If you want, you could rename the node to "keys" to follow the "generic
names recommendation" of the Device Tree V0.2 Spec Section
"2.2.2 Generic Names Recommendation."